test/TestApps/PushNotificationsTestAppPackage/Package.appxmanifest

```diff
@@ -49,7 +49,7 @@
 			</uap:Extension>
 			<com:Extension Category="windows.comServer">
 				<com:ComServer>
-					<com:ExeServer Executable="PushNotificationsTestApp\PushNotificationsTestApp.exe" DisplayName="SampleBackgroundApp" Arguments="-ReunionPushServer">
+					<com:ExeServer Executable="PushNotificationsTestApp\PushNotificationsTestApp.exe" DisplayName="SampleBackgroundApp" Arguments="----ms-protocol:ms-encodedlaunch://App/?ContractId=Windows.Push">
```

Comment:
This is an awfully long string for the developer to copy paste. Any chance we can alias this to something shorter and more friendly?